daemon/cmld.h

```diff
@@ -48,6 +48,11 @@
 #define LOGFILE_DIR "/data/logs"
 #endif
 
+// TODO Filename
+#define PROVISIONED_FILE_NAME "_cml_provisioned_"
+#define PROVISIONED_FLAG "1"
+#define NOT_PROVISIONED_FLAG "0"
```

Comment:
no need for those flags, see comment above.

Reply:
Done